h2o-r/tests/runitUtils/utilsR.R

```diff
@@ -1024,12 +1024,12 @@ assertCorrectSkipColumns <-
             else if (is.numeric(f1R[rind, ind]) || is.factor(f1R[rind, ind])) {
               if (allFrameTypes[ind] == 'time')
                 expect_true(
-                  abs(f1R[rind, ind] - f2R[rind, skipcount]) < 10,
+                  abs(f1R[rind, ind] - f2R[rind, skipcount]/1000) < 10,
```

Comment:
still, how do you justify that the 2 frames don't have the same value here?
both are the result of a POST to the backend (different API though), and of a call to `as.data.frame` on the payload.

Reply:
@sebhrusen Good point, thanks. It turns out this fix was wrong. The issue is that in the test we have 2 frames: `f1` and `f2` and we convert `f1` directly to dataframe and `f2` created by taking the original `f1` saving it to CSV[1] and then loading it and converting to data frame.  And this makes it lose the type information about the date time columns.

At least that's my hypothesis for now. (it works locally)

[1] or some other formats but in the CSV case we lose the type information about the date time  columns